modules/fb_instant_articles_api_rules/fb_instant_articles_api_rules_defaults.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Provide default rules integration to manage facebook instant article using API.
+ */
+
+/**
+ * Implements hook_default_rules_configuration().
+ */
+function fb_instant_articles_api_rules_default_rules_configuration() {
+  //Handling add/update node article as published in FB Instant Articles library.
```

Comment:
Nitpick: space needed after opening comment.  Same for all other inline comments.